sites/en/intro-to-rails/setting_the_default_page.step

```diff
@@ -34,6 +34,7 @@ Rails.application.routes.draw do
 }
 
 explanation {
+  tip "Docker users should run rails commands in a terminal running `docker-compose run web bash`."
```

Comment:
This is fine but I wondered if it could be improved a little at least to signify the steps taken in order to hopefully make it clearer.

> Docker users will need to run `docker-compose run web bash` first, where they can then run Rails related commands.

Not sure what you prefer but if we do change it seems this message is used on other pages too.

Reply:
@javierjulio if we have time, I think it's worth it to include the docker option every time we run a rails command...but I don't know how time consuming that will be to implement.